code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -0,0 +1,703 @@
+      tags:  
+        - Application Artifacts 
+      summary: Uploads artifact  on an OP.                       
+      description: Artifact is a zip file containing  scripts and/or packaging files like Terraform or Helm which are required to create an instance of an application.
+      operationId: uploadArtifact
+      requestBody:
+        description: An application can consist of multiple components. App providers are allowed to define seperate artifacts for each component  
+                            or  they could define a consolidated artifact at application level. 
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ArtifactDetails'
+        required: true
+      responses:
+        '201':
+          description: Artifact uploaded successfully
+          content: 
+            application/json:
+              schema:
+                $ref:  '#/components/schemas/UploadedArtifact'                
+        '400':
+          $ref: '#/components/responses/400'
+        '401':
+          $ref: '#/components/responses/401'
+        '403':
+          $ref: '#/components/responses/403'
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorInfo'
+              example:
+                status: 409
+                code: CONFLICT
+                message: "Artifacf duplicated"
+        '500':
+          $ref: '#/components/responses/500'
+        '501':
+          $ref: '#/components/responses/501'
+        '503':
+          $ref: '#/components/responses/503'       
+
+  /artifact/{artifactId}:
+    delete:
+      tags:
+        - Application Artifacts
+      summary: Delete an artifact from an OP
+      description: Delete an artifact
+      operationId: deleteArtifact
+      parameters:
+        - name: artifactId
+          in: path
+          description: Identificator of the artifact to be deleted provided by the OP NBI once the uploading was successful
+          required: true
+          schema:
+            $ref: "#/components/schemas/ArtifactId"
+
+      responses:
+        '204':
+          description: Artifact deleted
+        '400':
+          $ref: '#/components/responses/400'
+        '401':
+          $ref: '#/components/responses/401'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '409':
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorInfo'
+              example:
+                status: 409
+                code: CONFLICT
+                message: "Artifact with a running application cannot be deleted"
+        '500':
+          $ref: '#/components/responses/500'
+        '503':
+          $ref: '#/components/responses/503'
+
+  /edgeCloudPlatform:
```

Comment:
This duplicates what is in the MEC Exposure and Experience Management API, is that correct?

Reply:
It tends to comply with this intent

“I can retrieve a list of the operator’s MECs and their status, ordering the results by location and filtering by status (active/inactive/unknown)” 5SED 5MEE EXRA
[Developer Intents - Provisioning Intents 1.](https://github.com/camaraproject/EdgeCloud/blob/53aaba1b2f5acc1ba6ecdc8d7d3873f0cc222699/documentation/SupportingDocuments/Harmonisation%20of%20APIs/describing%20and%20harmonising%20the%20Edge%20APIs.md#L85:~:text=%E2%80%9CI%20can%20retrieve%20a%20list%20of%20the%20operator%E2%80%99s%20MECs%20and%20their%20status%2C%20ordering%20the%20results%20by%20location%20and%20filtering%20by%20status%20(active/inactive/unknown)%E2%80%9D%205SED%205MEE%20EXRA)

Reply:
In the just submitted v0.2.0 of the API, `edge-cloud-nodes` is the name we have selected for this parameter oriented by the GSMA naming. It is open to discussion as the final objective is to have the same names for the Simple Edge Discovery and for this set of methods. It is important to follow CAMARA guidelines (reason why we avoid using telco acronyms like mec).